.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+name: Test
+
+on:
+  push:
```

Comment:
I wonder if we would really want this, from the perspective of spending credits?  Might be adequate to just run it for PRs to main.  But having it on individual pushes on other branches is certainly handy if credits are effectively free.

Reply:
@mzagaja